SolidCarRace/Race.cs

```diff
@@ -1,41 +1,26 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
 
 namespace SolidCarRace
 {
     public class Race
     {
-        private readonly int numberOfLaps;
+        private readonly int _numberOfLaps;
 #pragma warning disable CS8632 // The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
-        private readonly List<Car>? cars;
+        private readonly List<Car>? _cars;
```

Comment:
great idea making fields read only
Nullable reference is a good idea, to prepare the application for it.